packages/client-generator-ts/src/TSClient/common.ts

```diff
@@ -388,3 +388,17 @@ export const skip = runtime.skip
 
   return ''
 }
+
+function notSupportedOnBrowserConst(fnc: string, browser: boolean) {
+  if (browser) {
+    return `() => { throw new Error('${fnc} is unable to run in the browser') }`
+  }
+  return `runtime.${fnc}`
+}
+
+function notSupportedOnBrowserType(fnc: string, browser: boolean) {
+  if (browser) {
+    return `typeof ${fnc}`
```

Comment:
❓ I am not sure if this is actually the best idea on how to stub out the type. It'd be an "incorrect" type anyways if actually tried to be used so not sure how valuable/relevant this is. 🤔